docs/overview.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Overview
+description: This section provides a conceptual overview of Octopus Deploy.
+position: 0
+---
+
+Welcome!
+
+This section provides a conceptual overview of Octopus Deploy.
+
+As an Octopus user, you define the process for deploying your software. You specify the environments the software is deployed to and who on your team can deploy to which environments. For instance, you might want developers to deploy to the Dev environment, but not Test or Production, you might want the QA team to deploy to the Test environment, but not Dev or Production. Taking this approach means that even if different members of the team trigger deployments, the deployment process remains consistent and predictable.
```

Comment:
This read awkwardly on first glance: "You specify the environments the software is deployed to and who on your team can deploy to which environments."

Consider something like: "You specify the environments your software can be deployed to and can control how (and which) teams deploy to these environments."

Reply:
Thanks @thebigmoosey, I've updated that sentence to read:
You specify the environments the software is deployed to, and you specify which teams can deploy to these environments.
I simplified your suggestion a bit further as the next sentence explains it in more detail.